From foo@baz Thu Aug 27 15:59:39 PDT 2009
Date: Thu, 27 Aug 2009 15:59:39 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: hv: fix remaining style issue in ChannelInterface.h

It snuck in with the other typedef cleanups, sorry about that.

Cc: Hank Janssen <hjanssen@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/hv/ChannelInterface.h |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/staging/hv/ChannelInterface.h
+++ b/drivers/staging/hv/ChannelInterface.h
@@ -29,6 +29,7 @@
 
 void GetChannelInterface(struct vmbus_channel_interface *ChannelInterface);
 
-void GetChannelInfo(struct hv_device *Device, struct hv_device_info *DeviceInfo);
+void GetChannelInfo(struct hv_device *Device,
+		    struct hv_device_info *DeviceInfo);
 
 #endif /* _CHANNEL_INTERFACE_H_ */
